src/modules/FSXAProxyApi.spec.ts

```diff
@@ -46,47 +46,45 @@ describe('FSXAProxyAPI', () => {
       proxyApi = new FSXAProxyApi(baseURL, LogLevel.NONE)
     })
     it('should make a fetch request to retrieve given id', async () => {
-      fetchMock.mockResponseOnce(JSON.stringify({ test: 'name' }))
+      fetchMock.mockResponseOnce(JSON.stringify({ mappedItems: [{ test: 'name' }] }))
 
-      proxyApi.fetchElement({ id, locale })
+      await proxyApi.fetchElement({ id, locale })
 
       const fetchMockBody = JSON.parse(fetchMock.mock.calls[0][1]?.body as string)
-
       const actualRequestUrl = fetchMock.mock.calls[0][0]
       const expectedRequestUrl = `${baseURL}${FSXAProxyRoutes.FETCH_ELEMENT_ROUTE}`
-
       expect(expectedRequestUrl).toEqual(actualRequestUrl)
-
       const actualRequestId = fetchMockBody.id
-
       expect(id).toEqual(actualRequestId)
-
       const actualRequestLocale = fetchMockBody.locale
-
       expect(locale).toEqual(actualRequestLocale)
     })
 
-    it('should throw an not found error when the response is 404', async () => {
+    it('should throw an not found error when the response is 404', () => {
```

Comment:
This async was a relict from how the code was incorrectly tested previously. It is not necessary here, since we are not awaiting anything, but simply returning a promise through the expect statement at the end of the function. See https://jestjs.io/docs/asynchronous